static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -350,6 +341,12 @@ export function NewOnboardingSidebar({
       orderedBeyondBasicsTasks.indexOf(a.task) - orderedBeyondBasicsTasks.indexOf(b.task)
   );
 
+  const taskKeyForWaitingIndicator = useMemo(() => {
+    return [...sortedGettingStartedTasks, ...sortedBeyondBasicsTasks].find(
+      task => !taskIsDone(task) && !!task.SupplementComponent
+    )?.task;
+  }, [sortedGettingStartedTasks, sortedBeyondBasicsTasks]);
```

Comment:
if available and task is not done, we display 1x waiting for event indicator for the whole quick start